### PR TITLE
HB-logger

### PR DESCRIPTION
**Pull request by KostiantynVorobiov**

### CODE REVIEW

**Review by aleksandr-hrankin — APPROVED** [review]

**Comment by aleksandr-hrankin** [comment]
Good!

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
-target/*git
\ No newline at end of file
+target/*git
```
Comment (KseniiaMakarova) [inline .gitignore]:
```suggestion
target/*
```

File: src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 2]
```diff
@@ -6,21 +6,26 @@
...
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class ShoppingCartDaoImpl implements ShoppingCartDao {
+    private static final Logger logger = Logger.getLogger(ShoppingCartDaoImpl.class);
+
     @Override
     public ShoppingCart add(ShoppingCart shoppingCart) {
+        logger.info("Trying to add shopping cart");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java:20]: let's log the update operation as well

File: src/main/java/com/cinema/Main.java [hunk 3]
```diff
@@ -74,34 +76,43 @@ public static void main(String[] args) throws AuthenticationException {
         onic.setPassword("911");
         UserService userService = (UserService) injector.getInstance(UserService.class);
         authenticationService.register(onic.getEmail(), onic.getPassword());
-        authenticationService.login(onic.getEmail(), onic.getPassword());
-        System.out.println(userService.findByEmail(onic.getEmail()).get());
+        try {
+            authenticationService.login(onic.getEmail(), onic.getPassword());
+        } catch (AuthenticationException e) {
+            logger.error(e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: warn level would be more suitable
and let's add the informative message here

File: src/main/java/com/cinema/Main.java [hunk 4]
```diff
@@ -74,34 +76,43 @@ public static void main(String[] args) throws AuthenticationException {
...
 
         ShoppingCartService shoppingCartService =
                 (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
         authenticationService.register("emily@com", "12345");
-        User emily = authenticationService.login("emily@com", "12345");
+        User emily = null;
+        try {
+            emily = authenticationService.login("emily@com", "12345");
+        } catch (AuthenticationException e) {
+            logger.error(e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: same

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 5]
```diff
@@ -25,7 +25,7 @@ public ShoppingCart add(ShoppingCart shoppingCart) {
             transaction = session.beginTransaction();
             session.save(shoppingCart);
             transaction.commit();
-            logger.info("Added new order" + shoppingCart + " successfully");
+            logger.info("Added new order " + shoppingCart + " successfully");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java]:
```suggestion
            logger.info("Added new shopping cart " + shoppingCart + " successfully");
```

File: src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 6]
```diff
@@ -58,13 +58,15 @@ public ShoppingCart getByUser(User user) {
...
     public void update(ShoppingCart shoppingCart) {
+        logger.info("Trying to update shopping cart");
         Transaction transaction = null;
         Session session = null;
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
             session.update(shoppingCart);
             transaction.commit();
+            logger.info("Updated order " + shoppingCart + " successfully");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java]:
```suggestion
            logger.info("Updated shopping cart " + shoppingCart + " successfully");
```

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**